src/test/java/TestUtil.java

```diff
@@ -123,7 +133,8 @@ public static void mockSnowflakeStageWithLocation(String stageLocation)
     PowerMockito.mockStatic(SnowflakeClient.class);
     PowerMockito // Note: clobbers mocks for SnowflakeClient.executeStatement
         .when(SnowflakeClient.executeStatement(anyString(),
-                                               any(SnowflakeConf.class)))
+                                               any(SnowflakeConf.class),
+                                               anyString()))
```

Comment:
Can you try making this something like
```
SnowflakeClient.executeStatement(
  anyString(), 
  any(SnowflakeConf.class), 
  expectedSchema))
```
so that we can verify if the correct schema is being passed through during the unit tests? Though, I remember hitting tons of issues trying to get this mocking framework to do what I wanted, so it's not the end of the world if it doesn't work out.

Reply:
Or maybe there's a better way to verify that this mocked SnowflakeClient is getting called with the right schema.

Reply:
I'll see if I can find a good option

Reply:
I was able to use Matchers.eq() for this purpose.